client-src/default/utils/log.js

```diff
@@ -2,6 +2,16 @@ import { getLogger } from 'loglevel';
 
 export const log = getLogger('webpack-dev-server');
 
+const originalFactory = log.methodFactory;
+
+log.methodFactory = (methodName, logLevel, loggerName) => {
+  const rawMethod = originalFactory(methodName, logLevel, loggerName);
+
+  return (message) => {
+    rawMethod(`[WDS] ${message}`);
+  };
+};
```

Comment:
No need this:
1. It is potentially break interface (it is bad practice)
2. We should migrate on `webpack-log`
3. It should be supported as factory method (not overriding)